examples/nuxt3/plugins/script.ts

```diff
@@ -21,14 +22,24 @@ const script = {
 export default defineNuxtPlugin(() => {
   const addScript = ref(false)
 
-  useHead({
-    script: () => [
-      { children: configure },
-      addScript.value && script,
-    ].filter((s): s is typeof script => !!s),
+  let scope : EffectScope | null = effectScope()
+
+  scope.run(() => {
```

Comment:
just an example, I'd imagine most users wouldn't do this so the watchers will never be stopped?